rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.64.0"
+components = [ "rustfmt", "clippy" ]
```

Comment:
Is this necessary? I'd rather be able to use the latest toolchain locally. Also, doesn't this duplicate the Rust version in psakefile.ps1 and Dockerfile.manylinux?

Reply:
You can still use the latest with the toolchain file there. It makes loading the correct rust toolchain very simple for CI. If you set the rustup override you'll continue to use your global override. If we move the the toolchain file we can remove the other instances you mention and load data from the toolchain file.

Reply:
> You can still use the latest with the toolchain file there.

It doesn't seem like it... when I try this branch, cargo will automatically download Rust 1.64 and use it, even though I have 1.66 installed. That's what I'd like to avoid.